### PR TITLE
Enable Roslyn's non blocking code complete

### PR DESCRIPTION
**Pull request by vasily-kirichenko**
and update `Microsoft.CodeAnalysis.xxx` packages to version `2.0.0-rc3-61225-01`

See https://github.com/Microsoft/visualfsharp/issues/1825#issuecomment-267468134

I cannot test it on VisualFSharp.sln due to https://github.com/Microsoft/visualfsharp/issues/2098.

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Common/LanguageService.fs [hunk 1]
```diff
@@ -305,6 +318,13 @@ type internal FSharpLanguageService(package : FSharpPackage) as this =
         workspace.Options <- workspace.Options.WithChangedOption(NavigationBarOptions.ShowNavigationBar, FSharpCommonConstants.FSharpLanguageName, true)
         let textViewAdapter = this.Package.ComponentModel.GetService<IVsEditorAdaptersFactoryService>()
         
+        let blockForCompletionOption = 
+            PerLanguageOption<bool>(
+                "CompletionOptions", "BlockForCompletionItems", defaultValue = true,
```
Comment (dsyme) [inline vsintegration/src/FSharp.Editor/Common/LanguageService.fs]: Do we really want to use "true" here as the default?

Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/Common/LanguageService.fs]: No idea. As we pass `false` later, I don't think it matters.

Comment (dsyme) [inline vsintegration/src/FSharp.Editor/Common/LanguageService.fs]: ok, thanks.

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Comment by vasily-kirichenko** [comment]
It turns out VS RC2 does not compatible with Roslyn nuget packages RC3-xxx. I'll revert updating.

**Comment by KevinRansom** [comment]
@vasily-kirichenko 

Thank you for this and everything else you have done for the F# IDE.  we appreciate it.

Kevin

**Comment by Pilchie** [comment]
Tagging @brettfo.  Sounds like we'll need to create an rc3 release branch where we can update to these, or else I expect Find References will be broken for F# in RC3.

**Pull request merged**